api/bind.go

```diff
@@ -92,7 +93,8 @@ func Bind(r *http.Request, dest interface{}) error {
 	}
 
 	if r.Header.Get("Content-Type") == "application/json" {
-		decoder := json.NewDecoder(r.Body)
+		reader := bufio.NewReader(r.Body)
+		decoder := json.NewDecoder(reader)
```

Comment:
@anatolie-ssv one of the downsides of using buffers is that we're allocating new buffers on every request.

WDYT about using sync.Pool  to reuse the created bufio instances?

Reply:
added!